### ISSUE TITLE
fluct new may be broken?

### ISSUE THREAD

**Issue by r7kamura** [r7kamura/fluct#29]
https://twitter.com/yutackall/status/636445670346371073

**Comment by r7kamura** [comment]
```
$ npm i fluct -g
/usr/local/bin/fluct -> /usr/local/lib/node_modules/fluct/bin/fluct
fluct@0.3.2 /usr/local/lib/node_modules/fluct
├── open@0.0.5
├── config@1.15.0
├── commander@2.8.1 (graceful-readlink@1.0.1)
├── mkdirp@0.5.1 (minimist@0.0.8)
├── yazl@2.2.2 (buffer-crc32@0.2.5)
├── ejs@2.3.3
├── glob@5.0.14 (path-is-absolute@1.0.0, inherits@2.0.1, minimatch@2.0.10, once@1.3.2, inflight@1.0.4)
├── moment@2.10.6
├── amazon-api-gateway-client@0.1.3 (stackable-fetcher-aws-signer-v4@0.0.1, stackable-fetcher@0.3.0)
├── node-aws-lambda@0.1.5 (async@1.4.2, https-proxy-agent@1.0.0)
├── aws-sdk@2.1.47 (xmlbuilder@0.4.2, xml2js@0.2.8, sax@0.5.3)
└── express@4.13.3 (escape-html@1.0.2, merge-descriptors@1.0.0, array-flatten@1.1.1, methods@1.1.1, cookie@0.1.3, range-parser@1.0.2, utils-merge@1.0.0, cookie-signature@1.0.6, fresh@0.3.0, vary@1.0.1, content-type@1.0.1, path-to-regexp@0.1.7, parseurl@1.3.0, etag@1.7.0, content-disposition@0.5.0, serve-static@1.10.0, depd@1.0.1, qs@4.0.0, debug@2.2.0, on-finished@2.3.0, proxy-addr@1.0.8, finalhandler@0.4.0, send@0.13.0, type-is@1.6.7, accepts@1.2.12)
$ fluct -v
module.js:338
    throw err;
          ^
Error: Cannot find module '../lib/deploy_command'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/fluct/bin/fluct:4:21)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
```

**Comment by r7kamura** [comment]
I think the reason is I forgot to build `lib` dir before `npm publish`

**Comment by r7kamura** [comment]
Fixed at v0.3.3

```
$ npm i fluct -g
/usr/local/bin/fluct -> /usr/local/lib/node_modules/fluct/bin/fluct
fluct@0.3.3 /usr/local/lib/node_modules/fluct
├── open@0.0.5
├── config@1.15.0
├── yazl@2.2.2 (buffer-crc32@0.2.5)
├── commander@2.8.1 (graceful-readlink@1.0.1)
├── mkdirp@0.5.1 (minimist@0.0.8)
├── ejs@2.3.3
├── glob@5.0.14 (path-is-absolute@1.0.0, inherits@2.0.1, once@1.3.2, inflight@1.0.4, minimatch@2.0.10)
├── node-aws-lambda@0.1.5 (async@1.4.2, https-proxy-agent@1.0.0)
├── express@4.13.3 (escape-html@1.0.2, merge-descriptors@1.0.0, array-flatten@1.1.1, cookie@0.1.3, utils-merge@1.0.0, cookie-signature@1.0.6, methods@1.1.1, fresh@0.3.0, range-parser@1.0.2, vary@1.0.1, path-to-regexp@0.1.7, content-type@1.0.1, etag@1.7.0, parseurl@1.3.0, content-disposition@0.5.0, serve-static@1.10.0, depd@1.0.1, qs@4.0.0, finalhandler@0.4.0, debug@2.2.0, proxy-addr@1.0.8, send@0.13.0, on-finished@2.3.0, accepts@1.2.12, type-is@1.6.7)
├── amazon-api-gateway-client@0.1.3 (stackable-fetcher-aws-signer-v4@0.0.1, stackable-fetcher@0.3.0)
├── moment@2.10.6
└── aws-sdk@2.1.47 (xmlbuilder@0.4.2, xml2js@0.2.8, sax@0.5.3)
$ cd /tmp
$ fluct --help

  Usage: fluct [options] [command]


  Commands:

    d|deploy             Deploy actions to AWS
    l|deployments        List recent deployments
    g|generate <name>    Generate a new resource from <generator> (e.g. "action")
    n|new <name>         Generate a new application
    o|open               Open the production root URL in your browser
    r|routes             List all routes
    s|server [options]   Launch a web server

  Options:

    -h, --help  output usage information

$ fluct new foo
Created ./foo
Created ./foo/README.md
Created ./foo/.gitignore
Created ./foo/actions
Created ./foo/actions/.keep
Created ./foo/package.json
```
